docs/document/content/manual/sharding-jdbc/configuration/config-java.en.md

```diff
@@ -279,15 +279,22 @@ The implementation class of `ShardingStrategyConfiguration`, used to configure n
 
 #### KeyGeneratorConfiguration
 
-| *Name*            | *DataType*                   | *Description*                                                                               |
-| ----------------- | ---------------------------- | ------------------------------------------------------------------------------------------- |
-| column            | String                       | Column name of key generator                                                                |
-| type              | String                       | Type of key generator, use user-defined ones or built-in ones, e.g. SNOWFLAKE, UUID, LEAF_SEGMENT, LEAF_SNOWFLAKE |
-| props             | Properties                   | The Property configuration of key generators                                                 |
+| *Name*                   | *DataType*               | *Description*                                                |
+| ------------------------ | ------------------------ | ------------------------------------------------------------ |
+| column                   | String                   | Column name of key generator                                 |
+| keyGenerateAlgorithm     | KeyGenerateAlgorithm     | key generate algorithm                                       |
+
+#### KeyGenerateAlgorithm
```

Comment:
explain of KeyGenerateAlgorithm could be removed

Reply:
done